artifacts/camara-cloudevents/event-subscription-template.feature

```diff
@@ -0,0 +1,369 @@
+    Given a valid <xxx> subscription request body
+    And the request property "$.protocol" is set to "HTTP"
+    And the request property "$.sink" is set to "azerty"    
+    When the request "create<xxx>Subscription" is sent
+    Then the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_SINK"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription retrieval scenarios #####################################
+
+  @<xxx>_subscriptions_29_no_authorization_header_for_get_subscription
+  Scenario: No Authorization header for get subscription
+    Given header "Authorization" is not present
+    And path parameter "subscriptionId" is set to the identifier of an existing <xxx> subscription
+    When the request "retrieve<xxx>Subscription" is sent 
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_30_expired_access_token_for_get_subscription
+  Scenario: Expired access token for get subscription
+    Given the header "Authorization" is set to expired token
+    And path parameter "subscriptionId" is set to the identifier of an existing <xxx> subscription
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+    
+  @<xxx>_subscriptions_31_invalid_access_token_for_get_subscription
+  Scenario: Invalid access token for get subscription
+    Given the header "Authorization" set to an invalid access token
+    And path parameter "subscriptionId" is set to the identifier of an existing <xxx> subscription
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_32_get_unknown_<xxx>_subscription_for_a_device
+  Scenario:  Get method for <xxx> subscription with subscription-id unknown to the system  
+    Given the path parameter "subscriptionId" is set to a value not corresponding to any existing subscription
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response  code is 404
+    And the response property "$.status" is 404
+    And the response property "$.code" is "NOT_FOUND"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription list retrieval scenarios #####################################
+
+  @<xxx>_subscriptions_60_no_authorization_header_for_list_subscription

```

Comment:
align test numbering

- create
- get
- list
- delete

and final multievent case

Reply:
Done !